src/coreclr/tests/src/Interop/PInvoke/AsAny/AsAnyTest.cs

```diff
@@ -339,6 +339,13 @@ private static void RunAsAnyFieldTests()
 
     private static void RunBestFitMappingTests()
     {
+        if (System.Globalization.CultureInfo.CurrentCulture.Name != "en-US")
+        {
+            Console.WriteLine("Non english platforms are not supported");
+            Console.WriteLine("passing without running tests");
```

Comment:
Maybe also specify exactly what is not supported / being run? Otherwise, it seems this test will run a bunch of other tests, then print out that it is passing without running tests.